src/Common/Event_Automator/Plugin.php

```diff
@@ -84,6 +84,22 @@ class Plugin extends \tad_DI52_ServiceProvider {
 	 */
 	protected $container;
 
+	/**
+	 * Plugin constructor.
+	 *
+	 * @since TBD
+	 *
+	 * @param \tad_DI52_Container $container The container to use.
+	 */
+	public function __construct( \tad_DI52_Container $container ) {
```

Comment:
Not something we can necessarily fix on this PR; but we need to move away from `tad_DI52_` classes here completely.